app/src/main/java/com/example/soptseminar1/ProfileFragment.kt

```diff
@@ -0,0 +1,79 @@
+package com.example.soptseminar1
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.bumptech.glide.Glide
+import com.example.soptseminar1.databinding.FragmentProfileBinding
+
+class ProfileFragment : Fragment() {
+
+    private var position = FOLLOWER
+    private var _binding: FragmentProfileBinding? = null
+    private val binding get() = _binding ?: error("Binding이 초기화 되지 않았습니다.")
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentProfileBinding.inflate(layoutInflater, container, false)
+        initTransactionEvent()
+        initImage()
+        return binding.root
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+
+    private fun initTransactionEvent(){
+        val followerfragment = FollowerFragment()
+        val repositoryfragment = RepositoryFragment()
+
+        childFragmentManager.beginTransaction()
+            .add(R.id.main_fragment, followerfragment)
+            .commit()
+
+        binding.btnFollower.isSelected = true
+        binding.btnFollower.setOnClickListener{
+            val transaction = childFragmentManager.beginTransaction()
+            binding.btnFollower.isSelected = true
+            binding.btnRepo.isSelected = false
+            when (position) {
+                REPOSITORY -> {
+                    transaction.replace(R.id.main_fragment, followerfragment)
+                    position = FOLLOWER
+                }
+            }
```

Comment:
```suggestion
            when (position) {
                REPOSITORY -> {
                    binding.btnFollower.isSelected = true
                    binding.btnRepo.isSelected = false
                    transaction.replace(R.id.main_fragment, followerfragment)
                    position = FOLLOWER
                }
            }
```
팔로워 버튼을 눌렀을 때, 현재 position이 레포지토리라면 Fragment를 교체하는 로직으로 이해했습니다. 그렇다면, 버튼의 `isSelected` 속성을 바꿔주는 코드 또한 when 안에 넣어도 될 것 같습니다. :-)